### PR TITLE
[ui] fix for making back button work on pooling in different states

### PR DESCRIPTION
**Pull request by thomasdavis**

### CODE REVIEW

**Review by 59023g — APPROVED** [review]

**Review by ryardley — DISMISSED** [review]
I believe this also fixes a bug where the whole page was disappearing when you hit back and dont have funds in a pool.

File: ui/app/src/views/CreatePoolPage.vue [hunk 1]
```diff
@@ -200,6 +200,15 @@ export default defineComponent({
         selectedField.value = null;
       },
 
+      backlink: computed(() => {
+        let link = '/pool';
+        if (fromSymbol.value && connected && Number(aPerBRatioMessage.value) > 0 &&
```
Comment (ryardley) [inline ui/app/src/views/CreatePoolPage.vue:205]: Just had a thought here it might be better to try and just use `router.back()`

**Review by 59023g — CHANGES_REQUESTED** [review]
I just tested this again. Doesn't work: `/pool/cctoken -> /pool/add-liquidity/cctoken "click back" -> /pool`

However, I tried @ryardley 's suggestion and all the cases *I know* worked. I know there's a case I'm missing.

**Review by 59023g — APPROVED** [review]
Ship it!!

**Pull request merged**